### PR TITLE
add RR-08

### PR DESCRIPTION
**Pull request by RR-08**

### CODE REVIEW

File: content/participant/jatinkumar.md [hunk 1]
```diff
@@ -1 +1,5 @@
-
```
Comment (CodingSpecies) [inline content/participant/jatinkumar.md:1]: @RR-08 is this someone else's file, which you have fixed, or something else?

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: content/participant/jatinkumar.md (hunk 1 above)
Comment (RR-08) [reply content/participant/jatinkumar.md:1]: Yes I have accidentally fixed someone else's file. Do I need to revert back for my pr to count?

Comment (CodingSpecies) [reply content/participant/jatinkumar.md:1]: I think so. @RR-08

**Review by CodingSpecies — COMMENTED** [review]
@RR-08 This shows that you have deleted your file. Perhaps create a new pr with only your markdown file and image, use this for help #2

**Comment by RR-08** [comment]
I hope it's fixed now. I have reverted the last commit and brought the required changes.

**Review by CodingSpecies — APPROVED** [review]
## Thank you for contributing! This PR is successfully part of Hacktoberfest! 🎊

@RR-08 thanks for the changes

**Pull request merged**